tests/input/cpp/cpp17.cpp

```diff
@@ -0,0 +1,8 @@
+bool CompareGenomeByFeatureResults::clickOnLink(std::string const& inLink) {
+   auto const [sequence, type, firstPosition, lastPosition] = parseLink(inLink);
```

Comment:
Your Test here isn't really doing anything as there is already a space before the '['.

You could add more spaces and use `sp_cpp_before_struct_binding    = force` to remove them. What does `remove` do on this file, are there any cases where it would be useful?